tmt/tests/system-reinstall/test-system-reinstall-bootc.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Check environment
+printenv
+
+# This script only runs on Packit and gating environment
+if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
+    pwd
+    # change to hack directory
+    cd ../../hack
+
+    ARCH=$(uname -m)
+    # Get OS info
+    source /etc/os-release
+    case "$ID" in
+        "centos")
+            BASE="quay.io/centos-bootc/centos-bootc:stream${VERSION_ID}"
+            TEST_OS="${ID}-stream-${VERSION_ID}"
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /usr/local/bin
+            cp -r /usr/local/bin .
+            ;;
+        "fedora")
+            BASE="quay.io/fedora/fedora-bootc:${VERSION_ID}"
+            TEST_OS="${ID}-${VERSION_ID}"
+            if [[ "$VERSION_ID" == 44 ]]; then
+                TEST_OS="${ID}-rawhide"
+            fi
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /usr/local/bin
+            cp -r /usr/local/bin .
+            ;;
+        "rhel")
+            # OSCI gating only
+            BASE="images.paas.redhat.com/bootc/rhel-bootc:latest-${VERSION_ID}"
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /var/lib/tmt/scripts
+            cp -r /var/lib/tmt/scripts ./bin
+
+            CURRENT_COMPOSE_ID=$(skopeo inspect --no-tags --retry-times=5 --tls-verify=false "docker://${BASE}" | jq -r '.Labels."redhat.compose-id"')
```

Comment:
Bigger picture I want to do https://gitlab.com/redhat/centos-stream/containers/bootc/-/issues/1174#note_2775383139